### ISSUE TITLE
Check whether there is an url mapping problem in the editor

### ISSUE THREAD

**Issue by PhilippMatthes** [st-tu-dresden/inloop#270]
After commit https://github.com/st-tu-dresden/inloop/commit/f49afc2d8c7b4b09543c9330702dc6e7a3d99cd3 I experienced a momentary url mapping problem in my local version of the master branch. The purpose of this issue is to further investigate and remove possible sources of this problem. 

How to reproduce: Run the application with `make run`. Open the browser. Go to a task. If it still works, go to the "editor" tab. This is the location which produced a 404: "No FlatPage matches the given query.".

**Comment by PhilippMatthes** [comment]
https://github.com/st-tu-dresden/inloop/blob/f49afc2d8c7b4b09543c9330702dc6e7a3d99cd3/inloop/solutions/urls.py#L42

After further investigation, I found out that this is the source of our mapping problem. 
When opening e.g. `.../the-fibonacci-sequence/editor` and appending `/`, such that the resulting URL is `.../the-fibonacci-sequence/editor/`, a 404 is produced.

I managed to mitigate this issue by performing following changes:
```diff
-         r'^(?P<slug>[-\w]+)/editor$',
+         r'^(?P<slug>[-\w]+)/editor/$',
```

Now both requests are successful. I'll commit this on a separate branch and open a new PR.

**Comment by martinmo** [comment]
> How to reproduce: Run the application with make run. Open the browser. Go to a task. If it still works, go to the "editor" tab. This is the location which produced a 404: "No FlatPage matches the given query.".

I cannot reproduce this and I still don't understand where the request for the `.../editor/` URL with trailing slash comes from. The upload view doesn't have a trailing slash either and works.

**Comment by PhilippMatthes** [comment]
Will do.

**Comment by martinmo** [comment]
What's the status on this, did you find out something more? Otherwise, please close this issue. In the meantime, I will close the corresponding PR because it will be out of sync anyway.

**Comment by PhilippMatthes** [comment]
I tried to further reproduce this issue, but without success. It seems to me that this problem was effectively caused by the browser. However, I found that the [simple change in the mapping](https://github.com/st-tu-dresden/inloop/pull/277/files#diff-1d5dad02919a6d859cc9bcbbfc0f863aR42
) supplied by my PR was sufficient to avoid this issue. But you are right, until this issue occurs again and until we get some more detailed information on why it exactly happened, we should close this issue.
